packages/web-components/fast-components/src/select/select.pw.spec.ts

```diff
@@ -200,15 +200,15 @@ describe("FASTSelect", function () {
                                 "fast-select"
                             )) as ElementHandle<FASTSelect>;
 
-                            await this.page.exposeFunction("sendEvent", type =>
+                            await this.page.exposeFunction("sendEvent", (type: string) =>
                                 expect(type).to.equal(eventName)
                             );
 
                             await element.evaluate((node, eventName) => {
                                 node.addEventListener(
                                     eventName,
                                     async (e: CustomEvent) =>
-                                        await window["sendEvent"](e.type)
+                                        await (window as any)["sendEvent"](e.type)
```

Comment:
Since this is a test file, I assume `any` is okay-ish here. Feel free to propose something cleaner 👍